app/main.py

```diff
@@ -39,6 +39,8 @@
 CACHE_DIR = os.getenv('CACHE_DIR', '/data/')
 
 LABEL_TEMPLATE = os.getenv('LABEL_TEMPLATE', 'playlist.html')
+COLLECT_POSITION = os.getenv('COLLECT_POSITION', None)
+COLLECT_CLASSNAME = f'collect collect-{COLLECT_POSITION}' if COLLECT_POSITION else 'collect'
```

Comment:
Could you move this logic to `playlist.js` so `COLLECT_CLASSNAME` isn't necessary?

Reply:
I'd have to recalculate the classname in `_base.html` which I'd prefer not to do.
That being said, I can at least move this logic into a local scope to get rid of this global.

Let me know what you think.

Reply:
Yeah nice one.